tests/integration/targets/inventory_vmware_vm_inventory/aliases

```diff
@@ -1,5 +1,4 @@
 shippable/vcenter/group1
 cloud/vcenter
-destructive
 needs/target/prepare_vmware_tests
 zuul/vmware/vcenter_only
```

Comment:
Please remove  zuul/vmware/vcenter_only  from the list. This test only works with govcsim.

Reply:
Is this blocking merge right now? I would really like to get my hands on the collection with this merged, and get a version number for that.

Reply:
Ok, I can patch @Akasurde's PR, and ensure it lands today.

Reply:
Thanks, really appreciate it!

Reply:
Done. I will keep an eye on the PR until it get merged. Thanks @AlanCoding :-)